src/styles/partials/components/_icon-heading.scss

```diff
@@ -51,3 +51,9 @@ $icon-size: 30px;
   line-height: 29px; /* Manually calculated for this element in the browser */
   padding: 0 !important; /* overrides default h2 in a dark section */
 }
+
+@media screen and (max-width: $small-breakpoint) {
+  .dg_icon-heading {
+    letter-spacing: $base-spacing-unit * 2;
```

Comment:
@theilman7 @tmccoy529 @sgrg1 this was done because on small screens the text was causing overflow, in turn causing a horizontal scroll which we dont want...would a comment be helpful here?

Reply:
It definitely wouldn't hurt.

Reply:
added.